### ISSUE TITLE
ckeditor 5 not works with ipad iphone page scroll when press space bar

### ISSUE THREAD

**Issue by giugit** [ckeditor/ckeditor5#12993]
## 📝 Provide detailed reproduction steps (if any)

I have all updated at the last version of software and 

there is the issue i wrote in the object that make the plugin 

unfortunately not usable due to the great number of apple device.

Any suggestion o workaround about how I can resolve this problem.

Thank You

**Comment by giugit** [comment]
I made some more test 
on macbook air no problems with edge and chrome, problems only with safari
on iphone and ipad the problem is present with all browsers

**Comment by giugit** [comment]
I read some very old answer about similar problems clssified as apple problem so i made one other test with one other editor TinyMCE and here therea are not problems!!!! so is not a apple problem!

**Comment by giugit** [comment]
SOLVED:
It was hard but in the end I discovered the problem

At the beginning it seemed an incompatibility with jquery

In fact, transforming the code into vanilla seemed to work

But then checking better only happened to the Phone and Mac, the problem persisted on iPad

Continuing to investigate I discovered the line of code that created the problem, it was this

$ ('. Ck-Content'). Bind ("DomsubreeModified", Function ()

in the vanillla version

Document.getelementsbyClassname ("CK-Content") [0] .Addeventlistener ('DomsubreeModified', OnsubreeModified, False);

As mentioned, the problem was solved only for iPhone and Mac but not for iPad.

I went back to jquery version and I replaced the line with
 
$ ('. Ck-Content'). On ('Keyup', Function () {

And now the problem is solved on all devices.

I wrote this long explanation you never know that someone

may have this same problem and maybe it doesn't lose all the time

that I employed thanks
